src/partials/navlist-item.html

```diff
@@ -9,7 +9,13 @@
         {{#if
         item.external}} nav-link_external{{/if}}{{#if item.disabled}}
         disabled{{/if}}" {{#if item.external}} target="_blank"
-        aria-label="{{title}}" rel="noopener noreferrer" {{/if}}>
+        aria-label="{{title}}" rel="noopener noreferrer" {{/if}}
+        {{#unless item.external}}
+        {{#ifequal @root.page-name item.id}} aria-current="{{title}}"{{/ifequal}}
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current , the aria-current attribute has to contain a special keyword. In this case, I think `aria-current="page"` is what we are aiming for. All other menu entries need to have the `aria-current` attribute removed